### DISCUSSION TITLE
S-meter

### DISCUSSION THREAD

**Discussion by lb5va** [df8oe/UHSDR#1915] [General]
1, Is there no way to calibrate the s-meter?  (Cant find it.)

2, My radio show aprx 3s with no antennea. Even when AGC is turned down to zero.
    Is that normal???

73, lb5va, Stig

**Comment by df8oe** [comment]
Hi Stig,

go to Dispaly Menu --> dBm calibrate. This calibrates all SSI elements. I think your gain is too high so that you see S3 without signal. Normal is S1...S2.

vy 73
Andreas
